index.bs

```diff
@@ -520,18 +520,20 @@ General Subsample Encryption constraints {#subsample-encryption}
 
 Within protected samples, the following constraints apply:
 - Protected samples SHALL be exactly spanned by one or more contiguous subsamples.
-- all [=OBU Headers=] and associated [=obu_size=] fields SHALL be unprotected.
-- [=Temporal Delimiter OBUs=], [=Sequence Header OBUs=], and [=Frame Header OBUs=] SHALL be unprotected.
-- [=Metadata OBUs=] MAY be protected.
-- [=Tile Group OBUs=] and [=Frame OBUs=] SHALL be protected.
-
-- Subsamples SHALL NOT cross OBU boundaries, except for multiple consecutive unprotected OBUs which SHOULD be spanned by a single subsample, with [=BytesOfProtectedData=] set to 0. In other words, a subsample that spans a protected OBU SHALL NOT span any other OBU.
-
-- A large unprotected OBU whose data size is larger than the maximum size of a single [=BytesOfClearData=] field MAY be spanned by multiple subsamples with zero size [=BytesOfProtectedData=].
+	- Subsamples SHALL NOT cross OBU boundaries, except for multiple consecutive unprotected OBUs which SHOULD be spanned by a single subsample, with [=BytesOfProtectedData=] set to 0. In other words, a subsample that spans a protected OBU SHALL NOT span any other OBU.
```

Comment:
Does it mean that for a frame with:

| Temporal Delimiter OBU | Frame Header OBU | Tile Group OBU Header | Tile Group OBU tile |

It has to be put in two subsamples instead of one?

Subsample 1: clear = TD + FH, encrypted = 0
Subsample 2: clear = TG header, encrypted = TG tile

I think it is against CENC spec recommendation: 
Subsample encryption entries SHOULD be as compactly represented as possible. For example, instead of two entries with {15 clear, 0 protected}, {17 clear, 500 protected}, use one entry of {32 clear, 500 protected}.